### DISCUSSION TITLE
Any plan to host these module on public ACR?

### DISCUSSION THREAD

**Discussion by kulmam92** [Azure/ResourceModules#837] [Q&A]
Do you have any plan to publish these modules to public ACR?
[Create private registry for Bicep modules](https://docs.microsoft.com/en-gb/azure/azure-resource-manager/bicep/private-module-registry?tabs=azure-powershell)

I think it will be easier for people to use it since they don't need to download the code like the modules in a terraform registry.
https://registry.terraform.io/

**Comment by AlexanderSehr** [comment]
Hey @kulmam92 , sorry for the late reply. It is planned to publish modules to the registry once available. Actually it was just released 2 days ago with version 0.5 of Bicep. 
As a test run we already publish 2 modules (i.e. AvailabilitySet & VirtualNetworks) to the registry and plan to publish more going forward. It will however not be one quick move as the registry has certain structural requirements we have to cater to (manually) for each individual module.
